src/cocoa/toga_cocoa/window.py

```diff
@@ -222,7 +223,19 @@ def set_title(self, title):
         self.native.title = title
 
     def set_position(self, position):
-        pass
+        x, y = position
+        screen = self.native.screen
+
+        # If the window isn't visible, it won't be on a screen;
+        # don't try to set the position.
+        if screen is None:
+            return
```

Comment:
I’m not sure if we should just give up here. I haven‘t tested it yet but ˋscreenˋ will probably also be ˋNoneˋ if the window was previously dragged or positioned off–screen instead of being set to not visible.

it might be better to then take the primary screen as a reference.

Reply:
I guess we fall back to the main screen in that case? The main screen is the one with the keyboard focus.

Reply:
That could make sense. See my comment below on the current ambiguity about which screen to use.